tests/test_detect_env.py

```diff
@@ -52,3 +52,17 @@ def test_detect_active_interpreter_non_existent_path(
 
     with pytest.raises(SystemExit):
         detect_active_interpreter()
+
+
+def test_detect_active_interpreter_log_resolved(
+    tmp_path: Path, mocker: MockFixture, capsys: pytest.CaptureFixture[str]
+) -> None:
+    mocker.patch("pipdeptree._detect_env.os.environ", {"VIRTUAL_ENV": str(tmp_path)})
+    mocker.patch("pipdeptree._detect_env.Path.exists", return_value=True)
+
+    actual_path = detect_active_interpreter()
+
+    captured = capsys.readouterr()
+
+    assert actual_path.startswith(str(tmp_path))
+    assert f"Resolved Python: [{tmp_path!s}" in captured.out
```

Comment:
It looks like this should become an integration test since we now need to call `main()` instead of `detect_active_interpreter()`. I think we can just assert that the output `startswith()` the python interpreter message.

Reply:
@kemzeb moved test to `test_main.py`. I mocked subprocess